### PR TITLE
Remove unused arg from VertexFormat::generate

### PR DESCRIPTION
**Pull request by brendanzab**
Depends on gfx-rs/gfx-rs#660

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -29,5 +29,5 @@ path = "src/lib.rs"
 crate-type = [ "dylib" ]
 plugin = true
 
-[dev_dependencies.gfx]
-git = "https://github.com/gfx-rs/gfx-rs"
+[dev_dependencies]
+gfx = "*"
```
Comment (kvark) [inline Cargo.toml]: are you sure about this? we'll have to publish gfx more often.

File: src/vertex_format.rs [hunk 2]
```diff
@@ -259,15 +259,6 @@ impl ItemDecorator for VertexFormat {
                     explicit_self: None,
                     args: vec![
                         generic::ty::Literal(generic::ty::Path {
-                            path: vec!["Option"],
```
Comment (kvark) [inline src/vertex_format.rs:262]: niiice

File: Cargo.toml (hunk 1 above)
Comment (brendanzab) [inline Cargo.toml]: If you have your `../.cargo/config` set up it should not be a problem. Here is mine:

``` toml
paths = [ "gfx-rs/", "gfx_gl/", "gfx_macros/", "gfx_device_gl/", "gfx_examples/" ]
```

Comment (brendanzab) [inline Cargo.toml]: I can remove the commit though, and we can do it later if we decide to. Sorry.

Comment (kvark) [inline Cargo.toml]: That would be good, yeah. I still don't quite understand though. If you don't re-publish gfx after every dependent change, your copy will still compile (due to cargo overrides), but everyone else will be broken. So is it really good to override?

**Pull request merged**